scico/linop/radon_svmbir.py

```diff
@@ -11,41 +11,51 @@
 `svmbir <https://github.com/cabouman/svmbir>`_ package.
 """
 
+from typing import Optional
+
 import numpy as np
 
 import jax
 import jax.experimental.host_callback
 
+import scico.numpy as snp
+from scico.loss import WeightedSquaredL2Loss
+from scico.typing import JaxArray, Shape
+
+from ._linop import LinearOperator
+
 try:
     import svmbir
 except ImportError:
     raise ImportError(
         "Could not import svmbir, please refer to INSTALL.rst "
-        "for instructions on how to install the SVMBIR."
+        "for instructions on how to install it."
     )
 
-import scico.numpy as snp
-from scico.loss import WeightedSquaredL2Loss
-from scico.typing import JaxArray, Shape
-
-from ._linop import LinearOperator
-
 
 class ParallelBeamProjector(LinearOperator):
-    r"""Parallel beam projector based on svmbir."""
+    r"""Parallel beam projector based on svmbir.
+    Performs tomographic projections of an image at specified angles.
+    """
 
     def __init__(
         self,
         input_shape: Shape,
         angles: np.ndarray,
         num_channels: int,
+        is_masked: Optional[bool] = False,
```

Comment:
Also worth exposing the non-negativity option of svmbir here?

Reply:
I think `SVMBIRWeightedSquaredL2Loss` would be more appropriate for adding the non-negativity since non-negativity does not have any bearing on projection and back-projection. Let me know what you think.

Reply:
Yes, you're right - I didn't look closely and had thought my comment was with respect to that class.

Reply:
Exposed positivity to the user.